js/mediaView.js

```diff
@@ -481,6 +482,10 @@ class MediaView extends ComponentView {
     }
   }
 
+  onMediaVolumeChange(event) {
+    Adapt.trigger('media:volumeChange', this, this.mediaElement.muted, this.mediaElement.volume);
```

Comment:
How does this signify which media is changed?
You're passing `[view, muted, volume]`, would `[_id, muted, volume]` not be easier to use where consumed?

Reply:
Hey @oliverfoster, this is so whatever is on the receiving end has the options available to read more into the model data. Sometimes we'll want available access to the _id, title, lang, displayTitle, model type, etc.
https://github.com/adaptlearning/adapt-contrib-xapi/blob/8b375def92ba6da3eea14e710b734f1a8373c8b4/js/XAPI.js#L670
https://github.com/cgkineo/adapt-xAPI/blob/ea9400ecb5cf0c18487d04dbe5247f849f2c863d/js/statements/abstractStatementModel.js#L24

Reply:
Did you mean to pass the model instead of the view?
```suggestion
    Adapt.trigger('media:volumeChange', this.model, this.mediaElement.muted, this.mediaElement.volume);
```

Reply:
Ah, I see how that would be more efficient. Thanks :)